client/js/lounge.js

```diff
@@ -37,6 +37,22 @@ $(function() {
 		"/voice",
 		"/whois"
 	];
+	var handledTypes = [
```

Comment:
What if we made it an object and set each property to `true`, then testing would be `handledTypes[type]`, which does not require calling `indexOf`?